app/controllers/api/v1/application_controller.rb

```diff
@@ -23,8 +23,6 @@ class ApplicationController < ActionController::API
         )
       end
 
-      private
```

Comment:
This is now a public API. It will be use directly in controller. For example, [this](https://github.com/Goose97/google-scraper-ruby/pull/49/files#diff-7d00cc43aaf2e771dd8efea4a52134720dfdca8b73776eca01e7c55fd6408457R30)

Reply:
There seems to be a misunderstanding as having this method set as private does not prevent calling it from a subclass. See https://stackoverflow.com/questions/1565269/are-there-good-reasons-for-private-to-work-the-way-it-does-in-ruby/1565640#1565640

Reply:
Oh, I see. Thanks for pointing it out. Rebased and fixed